src/app/containers/ATIAnalytics/amp/index.test.jsx

```diff
@@ -11,16 +12,54 @@ describe('Amp ATI Analytics', () => {
     document.body.appendChild(container);
   });
 
-  it('should render amp-analytics with script tag', () => {
-    act(() => {
-      ReactDOM.render(<AmpATIAnalytics />, container);
-    });
+  const testCases = [
```

Comment:
In these tests personally ide prefer if you mocked `useContext` to return `env` that you set. Then mock `atiBaseUrl(env)`, expect it to have been called with `env` and then mock to return a fake value. This test tests the functionality of `getEnv()` and `atiBaseUrl()` alot

Reply:
Good idea, have done this